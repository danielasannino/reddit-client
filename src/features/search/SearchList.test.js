import { SearchList } from './SearchList';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
const { shallow } = require('enzyme');

const mockStore = configureStore([])
const store = mockStore()

describe('SearchList component', () => {

    it('should render with given thr state from redux store', () => {
        const wrapper = shallow(
            <Provider store={store}>
                <SearchList />
            </Provider>
        )
        expect(wrapper).toMatchSnapshot()
    })
})