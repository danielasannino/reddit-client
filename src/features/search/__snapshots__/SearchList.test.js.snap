// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchList component should render with given thr state from redux store 1`] = `ShallowWrapper {}`;
